trunner/harness/psh.py

```diff
@@ -72,7 +72,14 @@ def __call__(self, result: TestResult) -> TestResult:
         if self.cmd is not None:
             self.dut.send(self.cmd + "\n")
             try:
-                self.dut.expect(f"{self.cmd}(\r+)\n")
+                if len(self.cmd) <= 68:
+                    self.dut.expect(f"{self.cmd}(\r+)\n")
+                else:
+                    full_cmd = ""
+                    while self.cmd in full_cmd:
+                        self.dut.expect("(\r+)\n")
+                        full_cmd += self.dut.before
```

Comment:
Isn't this a mistake? Shouldn't it be something loop continuing until it finds the entire self.cmd (?). Also, in such a case, what would happen if what the DUT receives is not what is being passed?
```suggestion
                    full_cmd = ""
                    while self.cmd not in full_cmd:
                        self.dut.expect("(\r+)\n")
                        full_cmd += self.dut.before
```

Reply:
In such case there is a timeout.